cloudbridge/cloud/providers/gce/resources.py

```diff
@@ -176,66 +177,80 @@ def zones(self):
 
 
 class GCEFirewallsDelegate(object):
+    NETWORK_URL_PREFIX = 'global/networks/'
+    DEFAULT_NETWORK = 'default'
   
     def __init__(self, provider):
         self._provider = provider
         self._list_response = None
 
     @staticmethod
-    def tag_id(tag):
+    def tagnet_id(tag, network):
```

Comment:
"tagnet" doesn't immediately map to a concept I recognize. Is there a better name for this (it isn't used much in here, so it could be more verbose)? If not, please add a comment as to what a "tagnet" represents.